Core/include/Acts/Vertexing/IterativeVertexFinder.ipp

```diff
@@ -429,7 +437,7 @@ Acts::IterativeVertexFinder<vfitter_t, sfinder_t>::reassignTracksToNewVertex(
     for (auto tracksIter = tracksBegin; tracksIter != tracksEnd;) {
       // consider only tracks that are not too tightly assigned to other
       // vertex
-      if (tracksIter->trackWeight > m_cfg.cutOffTrackWeight) {
+      if (tracksIter->trackWeight > m_cfg.cutOffTrackWeightReassign) {
```

Comment:
As discussed on mattermost, we do not set the trackWeight in the FullBilloirVertexFitter. 
Unless the fitter is fixed and the track weights are updated according to https://cds.cern.ch/record/2670380/files/ATL-PHYS-PUB-2019-015.pdf page 5, this line is problematic as an user is free to set the reassign track weight cut from configuration but trackWeight == 1 always.

Reply:
`tracksIter->trackWeight > m_cfg.cutOffTrackWeight` just does not make any sense because the tracks below that threshold will be filtered out anyways

Reply:
Right, but `tracksIter->trackWeight > m_cfg.cutOffTrackWeightReassign` doesn't make too much sense either if `tracksIter->trackWeight == 1` always. One can set `cutOffTrackWeightReassign` to whatever value in the [0,1) interval obtaining always the same result.

Reply:
but that is a different problem right? if we ignore the fact that `trackWeight` is always `0` this is an improvement IMO because you can set a threshold that will actually trigger the reassignment

Reply:
Right. I agree. Perhaps issue a warning if `cutOffTrackWeightReassign` is  != 1 in the config passed to the code unti #2149 is fixed? 

For the rest looks good. 
General comment: Is the sigma of the gaussian density seeder used as constraint in the finding step?